src/pencilkit.cs

```diff
@@ -3,8 +3,9 @@
 //
 // Authors:
 //	TJ Lambert  <t-anlamb@microsoft.com>
+//	Whitney Schmidt  <whschm@microsoft.com>
 //
-// Copyright 2019 Microsoft Corporation All rights reserved.
+// Copyright 2020 Microsoft Corporation All rights reserved.
```

Comment:
Same for the year, you never remove, only add:

```suggestion
// Copyright 2019, 2020 Microsoft Corporation All rights reserved.
```